src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/AndroidLoginPage.java

```diff
@@ -48,7 +49,7 @@ public void typePassword(String password) {
     }
 
     @Override
-    public void selectMaleSex() {
```

Comment:
in general, we can create ENUM for genders (male, female) and use in this method
suggested name method - selectGender(Gender gender)